src/app/(auth)/layout.tsx

```
@@ -0,0 +1,15 @@
+
+export default async function AuthLayout({children,}: Readonly<{children: React.ReactNode;}>) {
+    
+    const resposne  = await getUser();
+    if(resposne?.user){
+        redirect("/");
+    }
+
+    return 
+    <>
+        {children}
+    </>;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)